apps/judicial-system/backend/src/app/messages/notifications.ts

```diff
@@ -198,7 +198,7 @@ export const notifications = {
       id:
         'judicial.system.backend:notifications.prosecutor_court_date_email.body_v1',
       defaultMessage:
-        '{scheduledCaseText}<br /><br />{courtDateText}<br /><br />{courtRoomText}<br /><br />{judgeText}{registrarText, select, NONE {} other {<br /><br />{registrarText}}}{sessionArrangements, select, PROSECUTOR_PRESENT {} NONE_PRESENT {<br /><br /> Krafa tekin fyrir án boðunar í þinghald.} other {<br /><br />{defenderText}.}}',
+        '{scheduledCaseText}<br /><br />{courtDateText}<br /><br />{courtRoomText}<br /><br />{judgeText}{registrarText, select, NONE {} other {<br /><br />{registrarText}}}{sessionArrangements, select, PROSECUTOR_PRESENT {} NONE_PRESENT {<br /><br /> <strong> Krafan verður tekin fyrir án boðunar í þinghald. </strong>} other {<br /><br />{defenderText}.}}',
```

Comment:
Please change the id to v2 so the default message updates correctly in Contentful